app/src/main/java/com/twilio/conversations/app/common/extensions/TwilioExtensions.kt

```diff
@@ -207,13 +208,17 @@ suspend fun Conversation.getMessagesBefore(index: Long, count: Int): List<Messag
     })
 }
 
-suspend fun Conversation.sendMessage(message: Message.Options): Message = suspendCoroutine { continuation ->
-    sendMessage(message, object : CallbackListener<Message> {
+suspend fun Conversation.UnsentMessage.sendMessage() = suspendCancellableCoroutine<Message> { continuation ->
```

Comment:
name it `Conversation.UnsentMessage.send()` plz. Th idea is to name all coroutine-wrappers the same as original methods. So they will appear in auto-complete in AndroidStudio. That's how developers could get know about them :)

Reply:
Removed it. Added other extensions Conversation.sendTextMessage() & Conversation.sendMediaMessage() instead
- to enable mocking.